gtk4/src/auto/functions.rs

```diff
@@ -126,7 +126,7 @@ pub fn css_parser_warning_quark() -> glib::Quark {
 
 #[doc(alias = "gtk_disable_setlocale")]
 pub fn disable_setlocale() {
-    assert_initialized_main_thread!();
+    assert_not_initialized!();
```

Comment:
Don't you have to add this macro now? I don't think we had that before or did we?

Otherwise looks good to me

Reply:
```
$ rg 'assert_not_initialized'
gtk4/src/rt.rs
33:macro_rules! assert_not_initialized {
```

Seems to live in rt.rs already (and builds fine for me locally), or did you mean something else?

Reply:
No, all good then. Thanks!